android/WALT/app/src/main/res/layout/fragment_front_page.xml

```diff
@@ -1,275 +1,249 @@
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This diff looks ugly, it's because I removed the unnecessary parent FrameLayout, which unindented every line. The main changes are changing "Clock Sync" to "Diagnostics", changing the icon to magnifying glass, and making the divider above the MIDI button hide if the MIDI button is hidden.